moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -85,8 +85,31 @@ struct JumpThreshold
 
   explicit JumpThreshold(double jt_revolute, double jt_prismatic) : JumpThreshold()
   {
-    revolute = jt_revolute;
-    prismatic = jt_prismatic;
+    revolute = jt_revolute;    // Radians
+    prismatic = jt_prismatic;  // Meters
+  }
+};
+
+/** \brief Struct for containing max_step for computeCartesianPath
+
+    Setting translation to zero will disable checking for translations and the same goes for rotation */
+struct MaxEEFStep
+{
+  double translation;  // Meters
+  double rotation;     // Radians
+
+  explicit MaxEEFStep() : translation(0.0), rotation(0.0)
+  {
+  }
+
+  explicit MaxEEFStep(double max_translation) : MaxEEFStep()
+  {
+    translation = max_translation;
```

Comment:
should be ``: trannslation(max_translation)``

Reply:
Don't forget then, to initialize rotation step too ;-)